code-studio/scripts/build-js.js

```diff
@@ -100,7 +102,30 @@ Promise.all([
   }
 
   if (commander.watch) {
-    console.log("Watching for changes...");
+    // Watching for JavaScript file changes is already set up by browserify/watchify on each bundle
+    console.log("Watching for .js file changes...");
+
+    // Use gaze to set a watcher for scss file changes
+    gaze('src/css/*.scss', function (err, watcher) {
+
+      console.log("Watching for .scss file changes...");
+
+      /* Uncomment this if you want to debug .scss watching
+
+      console.log("cwd: " + process.cwd());
+      var watched = this.watched();
+      Object.keys(watched).forEach(function(watchedDir) {
+        watched[watchedDir].forEach(function(watchedFile) {
+          console.log('Watching ' + watchedFile + ' for changes.');
+        });
+      });
+      */
+
+      this.on('changed', function (filepath) {
+        console.log(filepath + ' was changed, rebuilding CSS');
+        child_process.execSync('npm run build-css', {stdio:[0,1,2]});
```

Comment:
Passing `{stdio: 'inherit'}` would be easier to understand than `{stdio:[0,1,2]}` and [should be equivalent](https://nodejs.org/api/child_process.html#child_process_options_stdio).